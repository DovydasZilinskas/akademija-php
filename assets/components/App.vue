<template>
  <div>
    <h2 class="center">Contact Me</h2>
  </div>
</template>

<script>
export default {
  name: 'app',
};
</script>

<style scoped>
.wrapper {
  margin: 0 auto;
}
</style>
