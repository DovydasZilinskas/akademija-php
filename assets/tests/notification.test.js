import { mount } from '@vue/test-utils';
import App from '../components/Notification.vue';

describe('Mounted Notification', () => {
  const wrapper = mount(App, {
    propsData: {
      message: 'Message',
    },
  });

  it('renders the correct message', () => {
    expect(wrapper.find('.notification').text()).toContain('Message');
  });

  it('has close button', () => {
    expect(wrapper.find('span').text()).toContain('Ã—');
  });

  it('renders correctly', () => {
    const wrapper = mount(App);
    expect(wrapper.element).toMatchSnapshot();
  });
});
