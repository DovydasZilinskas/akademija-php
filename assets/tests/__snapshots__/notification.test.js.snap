// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mounted Notification renders correctly 1`] = `
<div
  class="notification"
>
  
  
  
  <span
    class="close"
  >
    Ã—
  </span>
</div>
`;
